final_task/README.md

```diff
@@ -1,3 +1,31 @@
-# Your readme here
-Some text.
-Checkout how to write this file using *markdown*.
+# RSS-READER
+
+### Command-line utility which receives RSS URL and prints results in human-readable format.
+
+**Example:**
+python rss_reader.py https://news.yahoo.com/rss - -limit 1
+
+**Output**:
+
+Feed:  Yahoo News - Latest News & Headlines
+
+Title:  Thanks to Trump, the U.S. hasn&#39;t admitted a single refugee since September
```

Comment:
It seems that you have encoding problem in your application.

Reply:
Fixed